test/dataset/test_speech_synthesis_dataset.py

```diff
@@ -9,9 +10,12 @@ def cut_set():
     return CutSet.from_json('test/fixtures/ljspeech/cuts.json')
 
 
-def test_speech_synthesis_dataset(cut_set):
+@pytest.mark.parametrize('transform', [None, Standardize])
```

Comment:
actually we should add a third argument `[Standardize]` in there to ensure that it also works with sequence arguments

Reply:
true, adding